joomla/joomla-cms
# [4.0] Add back button to the transitions and stages view button toolbars

### Summary of Changes
A few weeks ago the submenu was removed from the workflow and it is very  circuitous to switch from stages to transitions. This improves the workflow with some "back" buttons.

### Testing Instructions
Go to the stages/transitions view and check the back button on the top left toolbar
